lib/public_key/asn1/ECPrivateKey.asn1

```diff
@@ -12,13 +12,19 @@ IMPORTS
 
 -- FROM New PKIX ASN.1 [RFC5912]
 
-EcpkParameters FROM PKIX1Algorithms88;
+EcpkParameters FROM PKIX1Algorithms88
+
+Attributes FROM PKCS-8;
 
 ECPrivateKey ::= SEQUENCE {
   version        INTEGER,
-  privateKey     OCTET STRING,
+  privateKey     CurvePrivateKey,
```

Comment:
Why is ECPrivateKey.asn1 changed? The changes has no impact on the code it is still exactly the same SEQUENCE.

Reply:
At least to me it feels clearer what the OCTET STRING actually represents logically, this is what newer versions of ASN-1 specs for PKIX calls it too. (Bringing our specs up to date with those is another and bigger job).